packages/@cdktf/hcl2json/package.json

```diff
@@ -15,7 +15,8 @@
     "test:ci": "jest --ci",
     "jest-watch": "jest --watch",
     "package": "./package.sh",
-    "dist-clean": "rm -rf dist"
+    "dist-clean": "rm -rf dist",
+    "prebuild": "cp $GOROOT/misc/wasm/wasm_exec.js ./wasm/wasm_exec.js"
```

Comment:
Nit: Maybe we can wrap this in a script and throw an error if the goroot is not set

Reply:
done